app/editor/src/components/form/select/Select.tsx

```diff
@@ -93,8 +96,13 @@ export const Select = <OptionType extends IOptionItem>({
             onClear?.();
           }
         }}
-        data-for="main-tooltip"
+        data-for="select-tooltip"
         data-tip={tooltip}
+        onClick={() => {
```

Comment:
Ensures the tooltip is hidden when an option is selected

Reply:
Also note that `onBlur` is not triggered after selecting until the user click the screen again (a second time)